classes/Path.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace OpenCFP;
+
+class Path
```

Comment:
What do you think about marking this class as `final`?

Reply:
I have no problem with that. I didn't do it since we have no ```final``` classes anywhere else in the project.